### ISSUE TITLE
Can't use absolute urls for icons on ios

### ISSUE THREAD

**Issue by oscarni** [san650/ember-web-app#49]
When I specify icons with an absolute url in the manifest they render with a prefixed `/` for apple meta tags.

I have this in manifest.js:

```js
icons: [
    {
      src: 'https://website.com/image/upload/123/icon-192.png',
      sizes: '192x192',
      type: 'image/png'
    }
]
```

The manifest file would be correct:

```json
"icons": [
  {
    "src": "https://website.com/image/upload/123/icon-192.png",
    "sizes": "192x192",
    "type": "image/png"
  }
]
```

But the apple tags would look like this:

```
<link rel="apple-touch-icon" href="/https://website.com/image/upload/123/icon-192.png" sizes="192x192">
```

` href="/` the slash can't be at the beginning of the url.. 😕

**Comment by san650** [comment]
Thanks for reporting this, I'll try to fix it as soon as I have a bit of free time.

**Comment by san650** [comment]
@oscarni I've just released v1.3.1 with the fix. Thank you for tackling this!

See https://github.com/san650/ember-web-app/releases/tag/v1.3.1
